resolvconf/dns/resolvconf.go

```diff
@@ -4,6 +4,9 @@ import (
 	"regexp"
 )
 
+// DefaultIPv4Servers is the list of default ipv4 nameservers
+var DefaultIPv4Servers = []string{"8.8.8.8", "8.8.4.4"}
```

Comment:
Can you please consolidate this and defaultIPv4Dns (in resolvconf/resolvconf.go) into a single constant ?